internal/pkg/version/version.go

```diff
@@ -4,5 +4,19 @@
 // Package version holds variables for generating version information
 package version
 
+const (
+	// LegacyAppTemplateVersion is the version associated with the application template before we started versioning.
+	LegacyAppTemplateVersion = "v0.0.0"
+	// AppTemplateMinVersionAlias is the least version number available for HTTPS alias.
+	AppTemplateMinVersionAlias = "v1.0.0"
+	// AppTemplateMinVersionStaticSite is the minimum app version required to deploy a static site.
+	AppTemplateMinVersionStaticSite = "v1.2.0"
```

Comment:
since these are in the `version` package now, i wonder if we could drop the "version" from them? `version.LegacyAppTemplate`, `version.MinStaticSite`, etc